appserver/tests/appserv-tests/devtests/batch/run_test.sh

```diff
@@ -36,7 +36,7 @@ test_run(){
   echo "ANT_OPTS=${ANT_OPTS}"
   ant ${TARGET} | tee ${TEST_RUN_LOG}
   ${S1AS_HOME}/bin/asadmin stop-database
-  ${S1AS_HOME}/bin/asadmin stop-domain
+  ${S1AS_HOME}/bin/asadmin stop-domain --kill=true
```

Comment:
check if stop-domain fails without force or kill, then invoke the force or kill option.

Reply:
> Let me see if I understand what was done here...
> 
> In some cases you changed the script to first try "asadmin stop-domain"
> and if that fails to try "asadmin stop-domain --force=true".
> 
> In some other cases you change the script to just use "asadmin stop-domain --kill=true".
> 
> I'm not sure what the difference is between --force and --kill.
> Why do you use one in some places and the other in other places?
> Is there a strategy for which to use when?
> 
> This all seems like a reasonable workaround. I only wish we could determine
> why the simple "asadmin stop-domain" is failing to begin with.


@bshannon , 

Let me try to give inputs for open questions, please find my comments inline starts with 
"Rohit".
1): In some other cases you change the script to just use "asadmin stop-domain --kill=true".

Rohit: We have tried to stop domain using force that didn't work. Hence we have opted for kill option.

2): I'm not sure what the difference is between --force and --kill.

Rohit:
--force
Specifies whether the domain is forcibly stopped immediately.
--kill
Specifies whether the domain is killed by using the functionality of the operating system to terminate the domain process.

3): why the simple "asadmin stop-domain" is failing to begin with.

Rohit: We are investigating it using the server logs and will keep you posted on the same.

The latest CI job has recent changes and the results can be seen under 
https://jenkins.eclipse.org/glassfish/blue/organizations/jenkins/glassfish/detail/PR-22865/1/pipeline/

There are test failures for these stages which are not fixed by this change -in nucleus_admin_all,  ql_gf_full_profile_all, ql_gf_nucleus_all, ql_gf_web_profile_all.

Reply:
@gurunrao  I'll incorporate your comments... Thanks